Ghidra/Features/Base/src/main/java/ghidra/program/flatapi/FlatProgramAPI.java

```diff
@@ -29,6 +29,7 @@
 import ghidra.app.plugin.core.clear.ClearCmd;
 import ghidra.app.plugin.core.clear.ClearOptions;
 import ghidra.app.plugin.core.searchmem.RegExSearchData;
+import ghidra.app.script.GhidraScript;
```

Comment:
This was imported so that the GhidraScript `@see` and `@link` wouldn't be stupid long.